src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -1324,4 +1324,14 @@ private AdsDataProvider() {
         }
     };
   }
+
+  @DataProvider
```

Comment:
In https://github.com/Wikia/selenium-tests/pull/1612 we don't use dataProvider. Have you considered following pattern from that PR? DataProvider with only one item introduces in my opinion redundant layer of abstraction